instrumentation/aws_sdk/example/Gemfile

```
@@ -2,7 +2,10 @@
 
 source 'https://rubygems.org'
 
-gem 'aws-sdk'
+gem 'aws-sdk-core', '~> 3'
+gem 'aws-sdk-sns', '~> 1'
 gem 'opentelemetry-api'
 gem 'opentelemetry-instrumentation-aws_sdk'
```

We will have to set min gem versions to when the V3 Observability support/refactoring was released:

```suggestion
gem 'aws-sdk-core', '~> 3.203.0'
gem 'aws-sdk-sns', '~> 1.83.0'
gem 'opentelemetry-api'
gem 'opentelemetry-instrumentation-aws_sdk', '~> 0.7.0'
```